be/src/storage/rowset/segment_iterator.cpp

```diff
@@ -770,7 +806,7 @@ void SegmentIterator::_switch_context(ScanContext* to) {
     }
 
     if (to->_read_chunk == nullptr) {
-        to->_read_chunk = ChunkHelper::new_chunk(to->_read_schema, _opts.chunk_size);
+        to->_read_chunk = ChunkHelper::new_chunk(to->_read_schema, _opts.chunk_size + _opts.chunk_size / 4 + 1);
```

Comment:
It seems we could return a larger chunk to executor layer. which could cause Aggregator crash. So I want to revert this PR. @xiaoyong-z @chaoyli @trueeyu

Reply:
our Aggregator couldn't handle greater than `runtime_state->chunk_size()` Chunk